### PR TITLE
Delete unused imports

### PR DESCRIPTION
**Pull request by dlsrks1021**
## Title (required)

Delete unused imports

## Description (required)

Delete unnecessary imports for optimizing.

## Tests performed (required)

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,9 +16,9 @@ buildscript {
 
 allprojects {
     repositories {
+        jcenter()
```
Comment (whym) [inline build.gradle]: Thanks for working on cleanup. Removing unused imports makes sense but does this change in build.gradle make any difference?

Comment (whym) [reply build.gradle]: Do you mean you removed [the 'Update build.gradle' commit](https://github.com/commons-app/apps-android-commons/pull/1606/commits/259f8c05d7097496711915add7a4acbe2841fe3d) already locally? If so, please amend and update your public branch.

Comment (whym) [reply build.gradle]: @dlsrks1021 any update?

Comment (dlsrks1021) [reply build.gradle]: I removed 'Update build.gradle' commit and modify 'remove unused imports' commit to do not change build.gradle.

Comment (whym) [reply build.gradle]: Thanks! :+1:

**Pull request merged**